rma/models/rma_order_line.py

```diff
@@ -704,7 +721,12 @@ def _onchange_lot_id(self):
         product = self.lot_id.product_id
         if product:
             self.product_id = product
-            self.uom_id = product.uom_id
+
+    @api.depends("uom_id")
+    def _compute_uom_id(self):
+        for rec in self:
+            if rec.product_id:
+                rec.uom_id = rec.product_id.uom_id
```

Comment:
uom_id compute depends on "uom_id"? this doesn't look correct

Reply:
It is not right indeed, fixed, thanks

Reply:
Thanks! I will fix pre-commit in a different pr if needed.